### PR TITLE
#413 Unit testing for `plot_projection()`

### PR DESCRIPTION
**Pull request by Dylan-Celli**
#413 This is a new PR to resolve the merge conflicts in the notebook file

### CODE REVIEW

File: core/plotting/test/test_plots.py [hunk 1]
```diff
@@ -84,6 +84,22 @@ def get_plot_histograms_and_lagged_scatterplot1() -> pd.Series:
...
+        """
+        data = [
+            [1, 1, 0, 1],
+            [0, 1, 0, 1],
+            [0, 0, 1, 1],
+            [1, 1, 1, 1],
+            [1, 1, 1, 1],
+        ]
+        index = pd.date_range(start="2023-01-01", periods=len(data), freq="D")
+        df = pd.DataFrame(data,index=index)
```
Comment (DanilYachmenev) [inline core/plotting/test/test_plots.py:101]: run linter on all the files, it should fix some small moments, like here space is missing after comma `...data, index=...`

File: core/plotting/test/test_plots.py [hunk 2]
```diff
@@ -170,3 +186,28 @@ def test_plot_spectrum2(self) -> None:
...
+
+    def test_plot_projection1(self) -> None:
+        """
+        Smoke test for `plot_projection()`.
+
+        - `mode` is unspecified.
+        - `ax` is unspecified.
+        """
+        test_df = self.get_plot_projection1()         
+        cplmiplo.plot_projection(test_df, special_values=[0])
```
Comment (DanilYachmenev) [inline core/plotting/test/test_plots.py:198]: set `special_values` outside the func

**Review by DanilYachmenev — COMMENTED** [review]
lgtm

**Comment by gpsaggese** [comment]
Ops this slipped through the cracks and now has conflicts. Can you pls resolve?

**Comment by Dylan-Celli** [comment]
After I merged master in to the branch other files have been changed accordingly but now fail the fast test 
```
im_v2/ccxt/data/extract/test/test_ccxt_extractor.py::TestCcxtExtractor1::test_download_ohlcv_websocket_kline_is_not_present (0.01 s) FAILED [ 23%]
im_v2/ccxt/data/extract/test/test_ccxt_extractor.py::TestCcxtExtractor1::test_download_ohlcv_websocket_kline_is_not_present ERROR [ 23%]
```
How should this be resolved, my code has not changed so I believe that one of the files did not merge well.

**Comment by DanilYachmenev** [comment]
@Dylan-Celli currently this PR tries to change 44 files
probably branched out from a different master branch
could you pls move the changes related to this PR to a new one?

**Pull request closed without merging**